### PR TITLE
[CSLD-70] Description in Swagger-docs

### PR DESCRIPTION
**Pull request by denisshevchenko**

### CODE REVIEW

File: src/wallet-web-api-swagger/Description.hs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+-- | Descriptions for each endpoint, for Swagger-documentation.
+
+module Description where
+
+import           Universum
+
+import           Data.Text (Text)
```
Comment (neongreen) [inline src/wallet-web-api-swagger/Description.hs]: Universum exports `Text`.

File: src/wallet-web-api-swagger/Main.hs [hunk 2]
```diff
@@ -43,7 +45,18 @@ import           Pos.Wallet.Web             (CAddress, CCoin, CCurrency,
                                              CUpdateInfo, CWallet, CWalletAssurance,
                                              CWalletInit, CWalletMeta, CWalletRedeem,
                                              CWalletType, SyncProgress, WalletError,
-                                             walletApi)
+                                             walletApi,
+                                             TestReset, GetWallet, GetWallets,
```
Comment (neongreen) [inline src/wallet-web-api-swagger/Main.hs]: It'd be better to import `Pos.Wallet.Web` qualified and avoid this huge import list.

File: src/wallet-web-api-swagger/Main.hs [hunk 3]
```diff
@@ -43,7 +45,18 @@ import           Pos.Wallet.Web             (CAddress, CCoin, CCurrency,
...
+                                             UpdateWallet, DeleteWallet, ImportKey,
+                                             WalletRestore, NewWallet, IsValidAddress,
+                                             GetProfile, UpdateProfile, NewPayment,
+                                             NewPaymentExt, UpdateTx, GetHistory,
+                                             SearchHistory, NextUpdate, ApplyUpdate,
+                                             RedeemADA, RedeemADAPaperVend,
+                                             ReportingInitialized, ReportingElectroncrash,
+                                             GetSlotsDuration, GetVersion, GetSyncProgress)
+
+import           Description                -- All endpoints' description imported from here.
```
Comment (neongreen) [inline src/wallet-web-api-swagger/Main.hs]: This should be a qualified import too. (Unqualified implicit imports are forbidden by the style guide.)

**Comment by staenrey** [comment]
Grammar check complete

**Pull request merged**